ProjectLighthouse/PlayerData/Profiles/User.cs

```diff
@@ -225,6 +229,12 @@ public string Serialize(GameVersion gameVersion = GameVersion.LittleBigPlanet1)
         return LbpSerializer.TaggedStringElement("user", user, "type", "user");
     }
 
+    private string serializeCrossControlPlanet(GameVersion gameVersion)
+    {
+        if (gameVersion != GameVersion.LittleBigPlanet2) return "";
+        return LbpSerializer.StringElement("crossControlPlanet", this.PlanetHashCrossControl);
```

Comment:
this could probably go inside serializeEarth, if you think of a better implementation than what I have here feel free to use it
```c#
    private string serializeEarth(GameVersion gameVersion)
    {
        string planet = LbpSerializer.StringElement<string>
        (
            "planets",
            gameVersion switch
            {
                GameVersion.LittleBigPlanet2 => this.PlanetHashLBP2,
                GameVersion.LittleBigPlanet3 => this.PlanetHashLBP3,
                GameVersion.LittleBigPlanetVita => this.PlanetHashLBPVita,
                _ => "", // other versions do not have custom planets
            },
            true
        );
        if(gameVersion == GameVersion.LittleBigPlanet2) 
            planet += LbpSerializer.StringElement<string>("crossControlPlanet", this.PlanetHashCrossControl, true);
        return planet;
    }
```